### ISSUE TITLE
Pop-up dictionary customization

### ISSUE THREAD

**Issue by ChubYee** [FooSoft/yomichan#2192]
Is there a way to style how the dictionary looks? I'm trying to change it to make it's styling and looks similar to the right image
![image](https://user-images.githubusercontent.com/104586776/176021638-dc0a3e79-bee6-488a-a467-4cafaa82ad98.png)

**Comment by ChubYee** [comment]
Is it possible to implement something to where the dictionary words and meanings aren't stacked on top of one another when scrolling/viewing, so that only one definition appears at a time with each mouse wheel scroll, since I find that it takes longer to process the information when it's cluttered together 

https://user-images.githubusercontent.com/104586776/176041373-f2bc69f5-59b7-4a2d-9412-a00b4462512e.mp4


https://user-images.githubusercontent.com/104586776/176040925-ec14559d-be98-4e3b-8176-4a4783956276.mp4

**Comment by Ceynou** [comment]
> Is it possible to implement something to where the dictionary words and meanings aren't stacked on top of one another when scrolling/viewing, so that only one definition appears at a time with each mouse wheel scroll? Since I find that it takes longer to process the information when it's cluttered together
> 8mb.video-Cgl-KuE2ghQr.mp4
> 176039948-adecacef-3efb-42cb-9864-8b0f5085b6dc.mp4

I don't think so, and use a good freq list like bccwj to sort by most used reading
optionally just follow the entirety of one these guide
[TheMoeWay](https://learnjapanese.moe/yomichan/)
[Rentry's guide](https://rentry.co/lazyXel)



> Is there a way to change how the dictionary looks? I'm trying to change it to make it's styling and looks similar to the right image ![image](https://user-images.githubusercontent.com/104586776/176021638-dc0a3e79-bee6-488a-a467-4cafaa82ad98.png)

you can mess with the popup apparence settings to get a repeated and redundant tag...
otherwise I think the rest is custom css

**Comment by toasted-nutbread** [comment]
> Is there a way to change how the dictionary looks? I'm trying to change it to make it's styling and looks similar to the right image

You can do some configuration using custom CSS rules in the settings, but some of the things shown in your second image aren't possible. Mainly the pitch accents.

> Is it possible to implement something to where the dictionary words and meanings aren't stacked on top of one another when scrolling/viewing, so that only one definition appears at a time with each mouse wheel scroll? Since I find that it takes longer to process the information when it's cluttered together

I don't believe this is currently possible using only CSS.
